core/src/main/java/org/apache/calcite/plan/RelOptMaterializations.java

```diff
@@ -60,7 +66,8 @@
    *         materialized views used in the transformation.
    */
   public static List<Pair<RelNode, List<RelOptMaterialization>>> useMaterializedViews(
-      final RelNode rel, List<RelOptMaterialization> materializations) {
+      final RelNode rel, List<RelOptMaterialization> materializations,
+      List<SubstitutionVisitor.UnifyRule> materializationRules) {
```

Comment:
There is no javadoc for the new parameter.

Reply:
ok, update the javadoc.